Source/Foundation/bsfUtility/Math/BsQuaternion.h

```diff
@@ -360,15 +348,19 @@ namespace bs
 		/** Gets the shortest arc quaternion to rotate this vector to the destination vector. */
 		static Quaternion getRotationFromTo(const Vector3& from, const Vector3& dest, const Vector3& fallbackAxis = Vector3::ZERO);
 
-		static const float EPSILON;
+		static constexpr const float EPSILON = 1e-03f;
 
 		static const Quaternion ZERO;
 		static const Quaternion IDENTITY;
 
-		float x, y, z, w; // Note: Order is relevant, don't break it
+		float x = 0.0f;
```

Comment:
For Quaternion, and a few other classes, like Vector, we don't want default value initialization. They can be used in very performance critical code where these values might get changed right away, so it ends up being extra instructions. These classes generally have specialized constructors that accept BS_ZERO or similar, that allow the user to zero-initialize if wanted.